src/controllers/articleController.js

```diff
@@ -447,6 +449,53 @@ class ArticleController {
       throw (err);
     }
   }
+
+  /**
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} returns an object containing a commented article response
+   */
+  static async commentArticle(req, res) {
+    const commenter = req.userData.id;
+    const slugId = req.params.articleSlug;
+    const { comment } = req.body;
+
+    const authorProfile = await User.findOne({ where: { id: commenter } });
+
+    const findArticle = await Article.findOne({ where: { slug: slugId } });
+
+    if (!findArticle) {
+      return res.status(404).json({
+        message: 'Article not found'
+      });
+    }
+
+    const { id } = findArticle;
+    const { userName, bio, image } = authorProfile;
+
+    const articleComment = await Comment.create({
+      articleId: id,
+      articleSlug: slugId,
+      userId: commenter,
+      comment: comment.trim(),
+    });
+
+    return res.status(201).json({
+      message: 'You have commented on the article',
+      articleComment: {
+        id: articleComment.id,
```

Comment:
looks like a opportunity to use the spread operator here

Reply:
How or why? I have never used it before!